website/docs/cookbooks/gh-action.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Use Scala CLI in GitHub Actions
+sidebar_position: 9
+---
+
+## Preparing simple aplication
+
+`scala-cli` lets you run, test, and package Scala code in various environments, also in GitHub CI. 
+To use Scala CLI features in a simple way you can use the GitHub Actions [scala-cli-setup](https://github.com/VirtusLab/scala-cli-setup) that installs everything necessary to run the Scala application and more.
+
+For example, here's a simple `ls` application to printing the files in the given directory:
+```scala title=Ls.scala
+//> using scala "2.13"
+//> using lib "com.lihaoyi::os-lib:0.7.8"
+
+@main def hello(args: String*) =
+  val path = args.headOption match
+    case Some(p) => os.Path(p, os.pwd)
+    case _       => os.pwd
+
+  if (os.isDir(path)) println(os.list(path).mkString(","))
+  else System.err.println("Expected directory path as a input")
+```
+
+and some tests for `ls` application:
+
+```scala title=TestsLs.test.scala
+//> using lib "org.scalameta::munit::0.7.27"
+import scala.util.Properties
+
+class TestsLs extends munit.FunSuite {
+  test("ls") {
+    // prepare test directory
+    val tempDir = os.temp.dir()
+    // create files
+    val exptectedFiles = Seq("Ls", "Hello").map(tempDir / _)
+    exptectedFiles.foreach(os.write(_, "Hello"))
+
+    // check
+    val scalaCLILauncher = if(Properties.isWin) "scala-cli.bat" else "scala-cli"
+    val foundFiles =
+      os.proc(scalaCLILauncher, "Ls.scala", "--", tempDir).call().out.text().trim
+
+    exptectedFiles.map(_.toString).foreach { file =>
+      assert(foundFiles.contains(file))
+    }
+  }
+}
+
+```
+
+## Run tests in Github CI
+
+The following configuration of `ci.yml` contains a definition of job that run tests using `scala-cli` for every platform defined in `matrix.OS`.
+
+```yaml
+jobs:
+  build:
+    runs-on: ${{ matrix.OS }}
+    strategy:
+      matrix:
+        OS: ["ubuntu-latest", "macos-latest", "windows-latest"]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        submodules: true
```

Comment:
I don't think we need this one (it's a bit superfluous here, the example app doesn't use git submodules). 
```suggestion
```

`fetch-depth` could be removed, but it's useful if users publish with Scala CLI, so we might as well keep this one.